examples/medplum-demo-bots/src/slack-bots/README.md

```diff
@@ -0,0 +1,77 @@
+# Medplum Slack Bot Integration
+
+This project integrates Medplum with Slack, allowing automatic notifications in a Slack channel when certain events occur in Medplum. It uses a Medplum Bot to listen for events and sends corresponding messages to a specified Slack channel.
+
+## Prerequisites
+
+- A Medplum account with bot creation permissions
+- A Slack workspace where you have permissions to create apps
+
+## Setup Instructions
+
+### 1. Slack App Setup
+
+1. Go to [Slack API](https://api.slack.com/apps) and click "Create New App".
+2. Choose "From scratch", give your app a name, and select your workspace.
+3. Under "Add features and functionality", select "Bots".
+4. Go to "OAuth & Permissions" in the sidebar.
+5. Under "Scopes", add the following Bot Token Scopes:
+   - `chat:write`
+   - `channels:read`
+6. At the top of the "OAuth & Permissions" page, click "Install to Workspace".
+7. After installation, you'll see a "Bot User OAuth Token". Save this token; you'll need it later.
+
+### 2. Medplum Bot Setup
+
+1. Log in to your Medplum account.
+2. Navigate to the Bot creation page.
+3. Create a new bot or select an existing one to edit.
+4. In the bot's code editor, paste the contents of the `demo-slack-bot.ts` file.
+5. Save the bot code.
+
+For further reference on creating a bot see [Medplum Demo Bots](https://github.com/medplum/medplum/tree/main/examples/medplum-demo-bots)
+
+### 3. Configure Bot Secrets
+
+1. In the Medplum bot configuration page, find the "Secrets" section.
```

Comment:
This is a bit misleading. The secretes are not really in the "Bot" configuration, they're in the Project configuration. That is, they are set once per project and shared across Bots. 

See this doc for more details: https://www.medplum.com/docs/access/projects#settings

Reply:
@rahul1 I've updated this part to refer to project level secrets but still point to https://www.medplum.com/docs/bots/bot-secrets for further reference.